src/main/java/com/tasks/taskapi/service/TaskService.java

```diff
@@ -16,6 +16,10 @@ public class TaskService {
 
     public List<Task> getAllTasks() { return taskRepository.getAllTasks(); }
 
+    public Task getTaskById(int id) {
+        return taskRepository.selectTaskById(id);
```

Comment:
can you call the method in taskservice the same as the method in taskrepository if thats ok xxx ntd only minor <3